docs/Configuration/yaml-files.md

```diff
@@ -224,7 +224,7 @@ The `controls` section allows you to configure scripts and device management (MD
 
 Controls for hosts that are in "No team" can be defined in `default.yml` or in `teams/no-team.yml` (but not in both files).
 
-- `scripts` is a list of paths to macOS, Windows, or Linux scripts.
+- `scripts` is a list of paths to macOS, Windows, or Linux scripts. Scripts used in No Team policy automations must be defined in `teams/no-team.yml`.
```

Comment:
@noahtalerman Script policy automation works the same as installer automation in that the item being automated has to be defined for the team before being used in an automation. Under the hood, using software installers as an example, this is because we need to upload the installer first before passing an ID back to configure the automation.

The implementation for the script side follows the same pattern: the script gets defined and uploaded, and then the uploaded script's ID gets associated with the policy to add the automation. This matches the behavior in the UI as well (you add a script in Controls -> Scripts, then point the policy at the script in the automation modal.

Of note, there's some extra indirection on installers that we don't need for scripts:

1. I believe there's a reference to software title rather than package for installers
2. Installers have their own YAML file (`package_path`) to define URL (and self-service flag), where scripts are referred to directly (via `path`).

Reply:
> Script policy automation works the same as installer automation in that the item being automated has to be defined for the team before being used in an automation.

Ah, I see. So the GitOps user has to add the software (and in this case script) to a team YAML (or "No team") they provide a relative path to the software/script file from their policies YAML.

Makes sense!

Reply:
Yep, that's correct.

Reply:
FYI I just updated to "No team" in quotes to be consistent w/ how we refer to "No team" in the docs.